modules/andes-core/broker/src/main/java/org/wso2/andes/server/cluster/coordination/hazelcast/HazelcastAgent.java

```diff
@@ -480,4 +496,16 @@ public void releaseInitializationLock() {
         }
     }
 
+    /**
+     * Method to check if the hazelcast instance has shutdown.
+     * @return boolean
+     */
+    public boolean isActive() {
+        if (null != hazelcastInstance) {
+            return hazelcastInstance.getLifecycleService().isRunning();
```

Comment:
Agreed. Fixed as suggested. The error will only be printed if the server is not in shutting_down state. If it is, the log is only printed in debug mode.